apps/coordinator/src/components/Wallet/fee-bumping/types.ts

```diff
@@ -0,0 +1,37 @@
+import { Network } from "@caravan/bitcoin";
+
+export interface UTXO {
```

Comment:
these are ultimately going to come from the fees package right?

Reply:
Yes ... unless I need a custom type to handle some changes